src/V3Error.h

```diff
@@ -131,6 +131,7 @@ class V3ErrorCode final {
         VARHIDDEN,      // Hiding variable
         WIDTH,          // Width mismatch
         WIDTHCONCAT,    // Unsized numbers/parameters in concatenations
+        PINNOTFOUND,    // A pin name in a module instance does not exist in the module being instanced
```

Comment:
Please insert in sorted order with other messages.  Slightly shorter comment so fits in 90 chars.

Add to pretendError as usually this will be an error.